### PR TITLE
*remove RxJava 1. Update libs

### PR DESCRIPTION
**Pull request by Null45**
RxJava 1 removed. Please release new artifact to jcenter

### CODE REVIEW

File: leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java [hunk 1]
```diff
@@ -1,99 +1,105 @@
 package com.schibstedspain.leku.geocoder;
 
+import android.annotation.SuppressLint;
```
Comment (ferranpons) [inline leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java]: Is it really needed?

File: leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java [hunk 2]
```diff
@@ -1,99 +1,105 @@
...
-    this.view = geocoderViewInterface;
-  }
-
-  public void stop() {
-    this.view = nullView;
-    if (compositeDisposable != null) {
-      compositeDisposable.clear();
+    private static final int RETRY_COUNT = 3;
+
+    private GeocoderViewInterface view;
```
Comment (ferranpons) [inline leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java]: Why you changed the line space of the whole class? It's confusing for reviewing it.

File: leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java (hunk 1 above)
Comment (Null45) [reply leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java]: Just coz getLastKnownLocation() in new versions of https://github.com/mcharmas/Android-ReactiveLocation library annotated with 
```
    @RequiresPermission(
            anyOf = {"android.permission.ACCESS_COARSE_LOCATION", "android.permission.ACCESS_FINE_LOCATION"}
    )
```

File: leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java (hunk 2 above)
Comment (Null45) [reply leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java]: Just changed tabs to spaces. U can just change my previous commit If some code style is wrong in your opinion

Comment (ferranpons) [reply leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java]: But that makes checkstyle fail. You would need to revert that change and only modify the lines needed.

File: leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java (hunk 1 above)
Comment (ferranpons) [reply leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java]: We would need to review that. But it's ok for now 👍

**Pull request merged**